deploy/crds/postgresql_v1alpha1_database_cr.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: postgresql.baiju.dev/v1alpha1
 kind: Database
 metadata:
   name: demo-database
-  namespace: openshift-operators
+  namespace: demo-app
```

Comment:
What's the meaning of `demo-app` hardcoded as namespace? Would it apply to have a `REPLACE_NAMESPACE` following the other variable names?

Reply:
This `deploy/crds/postgresql_v1alpha1_database_cr.yaml` is just an example CR, isn't it?
The operator can create CRs in any namespace - so `demo-app` is an example.

On the other hand, the namespace used at `REPLACE_NAMESPACE` by the `Makefile` is the `Quay.io`'s application namespace - basically quay username or org.